staker/stakerapp.go

```diff
@@ -827,22 +874,18 @@ func (app *App) checkTransactionsStatus() error {
 func (app *App) waitForStakingTxConfirmation(
 	txHash chainhash.Hash,
 	depthOnBtcChain uint32,
-	ev *notifier.ConfirmationEvent) {
+	ev *notifier.ConfirmationEvent,
+) {
 	defer app.wg.Done()
 
-	// check we are not shutting down
-	select {
-	case <-app.quit:
-		ev.Cancel()
-		return
-
-	default:
-	}
-
```

Comment:
Here since there is nothing between the select check and the for with select, I have removed 
but if this is not wanted, let me know